v0/destinations/salesforce/transform.js

```diff
@@ -39,20 +41,22 @@ async function getSFDCHeader(destination) {
   )}${encodeURIComponent(destination.Config.initialAccessToken)}&client_id=${
     destination.Config.consumerKey
   }&client_secret=${destination.Config.consumerSecret}&grant_type=password`;
-  let response;
-  try {
-    response = await axios.post(authUrl, {});
-  } catch (error) {
-    logger.error(error);
+  const salesforceAuthResponse = await httpPOST(authUrl, {});
```

Comment:
Do we need "salesforce" in the variable names? They are making them longer. If it's unavoidable, consider using at least the shortform "sf"?

Reply:
Make sense